docs/reference/query_types/parameters/common/query/limit.rst.inc

```diff
@@ -2,12 +2,13 @@
 ~~~~~~~~~
 
 Defines the maximum number of items to return.
+If ``null`` is used as a value, the limit won't be set.
```

Comment:
Similar as commented for offset, with `null` limit will default to `25`.

Reply:
Fixed in https://github.com/netgen/ezplatform-site-api/pull/187/commits/31e62e94e070fe6f8e0616ae050d6551fc4e2eb4